roles/rsyslog/handlers/main.yml

```diff
@@ -1,5 +1,10 @@
 ---
-- name: restart rsyslogd
+- name: reset failure
+  listen: restart rsyslogd
+  command: systemctl reset-failed rsyslog
```

Comment:
@richm, @pcahyna, I know this is not a right thing to do - we should not call systemctl in the command module directly, but by this resetting the failure in the repeated restarts by the following handler, the CI test failure [1] disappears.
[1]
```
   Apr 13 17:47:43 ibm-p8-kvm-03-guest-02 systemd[1]: rsyslog.service: Start request repeated too quickly.
   Apr 13 17:47:43 ibm-p8-kvm-03-guest-02 systemd[1]: rsyslog.service: Failed with result 'start-limit-hit'.
   Apr 13 17:47:43 ibm-p8-kvm-03-guest-02 systemd[1]: Failed to start System Logging Service. 
```